Directory.Build.props

```diff
@@ -9,7 +9,7 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <HealthcareSharedPackageVersion>2.1.40</HealthcareSharedPackageVersion>
+    <HealthcareSharedPackageVersion>3.0.6</HealthcareSharedPackageVersion>
```

Comment:
Can you update this to the latest of the shared package version? (I believe 3.0.10)

Reply:
I tried that locally, but looks like there had been some changes from 3.0.6 to 3.0.10 which needs some more changes. I would prefer to have a different PR for that. Thoughts?